### PR TITLE
Patch arrow version

### PR DESCRIPTION
**Pull request by edude03**
###### Motivation for this change
For ZHF: #122042 I noticed that it can't find arrow 0.17.0 and it looks like in [this change](https://github.com/NixOS/nixpkgs/commit/66ebf2b76d42885ca0cf57f66319df2948c91b59#diff-5d2214d585ec8b549f1bd00d407779465a755db49308f5ca7145c8999a34dca2) arrow 0.17.0 was removed and replaced with arrow 1. There is a separate `arrow_1` package so I do wonder if it's not better to revert the 0.17 -> 1 change but anyway, this should fix this one case.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/admin/oci-cli/default.nix [hunk 1]
```diff
@@ -46,7 +46,8 @@ python3Packages.buildPythonApplication rec {
       --replace "cryptography==3.2.1" "cryptography" \
       --replace "pyOpenSSL==19.1.0" "pyOpenSSL" \
       --replace "PyYAML==5.3.1" "PyYAML" \
-      --replace "six==1.14.0" "six"
+      --replace "six==1.14.0" "six" \
+      --replace "arrow==0.17.0" "arrow"
```
Comment (jonringer) [inline pkgs/tools/admin/oci-cli/default.nix:50]: To comply with [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#submitting-changes) please have the commit message name be of the format
```
<pkg-name>: <subject-line>
```
for more examples, please look at https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#submitting-changes

in your case, the commit message should be:
```
oci-cli: patch arrow version
```

**Review by jonringer — COMMENTED** [review]
otherwise LGTM

File: pkgs/tools/admin/oci-cli/default.nix (hunk 1 above)
Comment (edude03) [reply pkgs/tools/admin/oci-cli/default.nix:50]: Thanks! Done!

**Review by jonringer — APPROVED** [review]
LGTM
shows usage

```
https://github.com/NixOS/nixpkgs/pull/122105

1 package built:
oci-cli
```

**Pull request merged**